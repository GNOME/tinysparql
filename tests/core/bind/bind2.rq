PREFIX  : <http://example.org/ns#>
SELECT  ?a ?b ?x AS ?double
WHERE
    { ?a :integer ?b .
      BIND ((?b * 2) AS ?x) .
      FILTER (?x < 50) .
    }
ORDER BY ?a
