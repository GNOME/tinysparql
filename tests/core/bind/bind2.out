"http://example.org/ns#e"	"17"	"34"
